todoapp/src/components/TodoList.vue

```diff
@@ -2,44 +2,65 @@
 import { onMounted, ref, watch } from 'vue'
 import moment from 'moment'
 import type { TodoType } from '../types/text'
-
+import DeletePopUP from './DeletePopUP.vue'
 import Header from './Header.vue'
 import Todos from './Todos.vue'
 
 const todos = ref<TodoType[]>([])
+const popUp = ref(false)
+const selectedTaskIndex = ref()
 
 function addTodo() {
   const currentDate = moment().format('DD.MM.YYYY')
   const newTodo: TodoType = {
-    title: 'Pay for rent',
-    text: 'Lorem ipsum dolor sit amet ipsum dolor sit amet',
+    title: 'Add a title',
+    text: 'Add a description',
     priority: 0,
+    priorityChange: false,
     status: false,
-    created_at: ` ${currentDate}`
+    created_at: ` ${currentDate}`,
+    editing: false
   }
   todos.value.push(newTodo)
 }
-
 onMounted(() => {
   todos.value = JSON.parse(localStorage.getItem('todos')!) || []
```

Comment:
Please avoid as much as you can using "!" at the end of expressions this doesn't solve any error just say to TS compiler that you are sure that the value you try to access will exists. This is a good example why to not use this "trick" if in your localStorage doesn't exist the todos attribute you will receive an error because JSON cannot parse undefined
You can simple check if the `localStorage.getItem('todos')` exist and than parse